torch/jit/annotations.py

```diff
@@ -246,6 +246,8 @@ def try_ann_to_type(ann, loc):
         return TensorType.get()
     if ann is torch.Tensor:
         return TensorType.get()
+    if inspect.isclass(ann) and issubclass(ann, torch.Tensor):
```

Comment:
Why not change the case above to `if subclass(ann, torch.Tensor`? I think that covers `Tensor`, `Parameter` and all future subclasses of `Tensor`?